test/topology/test_aggregation.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+import asyncio
+import pytest
+import time
+import logging
+import random
+
+from cassandra.cluster import NoHostAvailable  # type: ignore
+
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error
+from test.pylib.util import wait_for, wait_for_cql_and_get_hosts
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_cancel_mapreduce(manager: ManagerClient):
```

Comment:
You need to add `@skip_mode` attribute, the test uses error injection which is disabled in release mode.